components/menu.module.css

```
@@ -101,6 +101,10 @@
   border-bottom: 1px solid white;
 }
 
+.menu li a:hover {
+  color: white;
+}
+
```

```suggestion
}

.menu li a {
  border-bottom: 0.5rem;
}

.menu li a:hover {
  color: white;
  border-bottom: 1px solid white;
}

```
The underline of each menu item is still longer than it needs to be, so I suggest fixing it this way.
Please check it on your own once again 🙏

---

Hi Kaho! Thank you for reviewing. I fixed underline length. please check review again.